feature/sponsor/src/main/java/io/github/droidkaigi/confsched2019/sponsor/ui/SponsorFragment.kt

```diff
@@ -88,18 +89,21 @@ class SponsorFragment : DaggerFragment() {
 
     private fun setupSponsorsLayout(sponsorCategories: List<SponsorCategory>) {
         val sponsors = sponsorCategories.map { category ->
-            category.toSection()
+            category.toSection(sponsorCategories.last() == category)
         }
         groupAdapter.update(sponsors)
     }
 
-    private fun SponsorCategory.toSection() = Section().apply {
+    private fun SponsorCategory.toSection(isLastItem: Boolean) = Section().apply {
         setHeader(HeaderItem(category.title))
         addAll(
             sponsors.map { sponsor ->
                 sponsor.toItem(category)
             }
         )
+        if (isLastItem.not()) {
```

Comment:
We do not use `.not()` in our current policy. 🙏 I think that it is okay to set up an issue if you are concerned.

Reply:
@takahirom 
Fixed not to use `.not`.  https://github.com/DroidKaigi/conference-app-2019/pull/424/commits/08b795ef4daa9874d020db6034d4a0411bd274b9
Please review again 🙇 

Can you tell me why policy is not using `.not`?

Reply:
There is no particularly big reason. I think that this is pretty much due to preference, such as hard to see if denial later comes. Right now we are unifying with a policy not to use .not.

Reply:
@takahirom 
I understood.
Thank you for answering. 😇